README.mdown

```diff
@@ -36,7 +36,7 @@ The course consists of six sessions, all of which have common features:
 
   The rest were spawned out of the brain of Mr Cheek himsef.
 
-  Don't forget to run "rack session_number:challenge_number" as you go along for feedback on whether your solutions are correct, and once you've got all your tests passing, you can compare your solution to the solutions in the 'solved' section (next)
+  Don't forget to run "rake session_number:challenge_number" as you go along for feedback on whether your solutions are correct, and once you've got all your tests passing, you can compare your solution to the solutions in the 'solved' section (next)
```

Comment:
@jordanpoulton Shouldn't "rack" be "rake" in this case to run the tests?

Reply:
Support Josué's comment with Makers' Rubykickstart ref.
https://github.com/makersacademy/ruby-kickstart/blob/master/README.mdown

Jordan (or Joshua) was probably thinking of a different kind of "rack", if ya know what I mean ;)